site/app/templates/forum/ThreadPostForm.twig

```diff
@@ -198,6 +198,26 @@
         $("input[name='title']").change(function () {
             $(this).val($.trim($(this).val()));
         });
+
+        $(".thread-announcement-checkbox").click(function (){
+            if($(".thread-announcement-checkbox").prop("checked")){
+                $(".thread-announcement-checkbox").prop("checked", true);
+                $(".pinThread-checkbox").prop("checked", true);
```

Comment:
you should also disable the checkbox here [https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input#htmlattrdefdisabled](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input#htmlattrdefdisabled)

Because if the pin+email box is checked you dont want to ever be able to uncheck the other pin button (because if the first checkbox is clicked the second one does nothing)

you should just be able to add this line after `$(".pinThread-checkbox").prop('disabled', true);`